docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -622,16 +653,19 @@ public class GreetingResourceTest {
 }
 ----
 
-This test code acquires a token using a `password` grant from the test `Auth0` domain which has an application with the client id `test-auth0-client` registered, and which has a user `alice` with a password `alice` created. The test `Auth0` application must have the `password` grant enabled for a test like this one to work. This example code also shows how to pass additional parameters. For `Auth0`, these are the `audience` and `scope` parameters.
-
+This test code acquires a token by using a `password` grant from the test `Auth0` domain, which has registered an application with the client id `test-auth0-client`, and created the user `alice` with password `alice`.
+For a test like this to work, the test `Auth0` application must have the `password` grant enabled.
+This example code also shows how to pass additional parameters.
+For `Auth0`, these are the `audience` and `scope` parameters.
 
 [[integration-testing-keycloak-devservices]]
 ==== Dev Services for Keycloak
 
-Using xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] is recommended for the integration testing against Keycloak.
-`Dev Services for Keycloak` will launch and initialize a test container: it will create a `quarkus` realm, a `quarkus-app` client (`secret` secret) and add `alice` (`admin` and `user` roles) and `bob` (`user` role) users, where all of these properties can be customized.
+Consider using xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] for integration testing against Keycloak.
```

Comment:
`Consider`  is losing the `recommended` message IMHO

Reply:
To avoid using "recommended", I could also just say "Use xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] for integration testing against Keycloak." Would this work?

Reply:
Yes, our style guides state that we are to avoid using such phrases as "we recommend" or "we suggest" and to instead direct users to _take_ the recommended action and avoid potential user uncertainty. For example, instead of saying "We recommend that you issue the XYZ command.", we should just say "Issue the XYZ command."

Reply:
Thanks  @sheilamjones, though I'm not sure we need to follow every single guideline, without it, it is just an instruction